.travis.yml

```diff
@@ -111,3 +111,50 @@ matrix:
         - libb2-dev
     before_install:
     - source ./.travis/install_cuda.sh
+
+  # Job: Clang tidy
+  - os: linux
+    env: V=1 PATH="/usr/bin:$PATH" NO_COMPILE=1 TEST=tidy
+    addons:
+      apt:
+        packages:
+        - elfutils
+        - libzstd1-dev
+        - libb2-dev
+    before_install:
+    - pip install --user compiledb
+
+  # Job: Check formatting
+  - os: linux
+    env: V=1 PATH="/usr/bin:$PATH" NO_COMPILE=1 TEST=check_format
+    addons:
+      apt:
+        packages:
+        - elfutils
+        - libzstd1-dev
+        - libb2-dev
+
+  # New Jobs go here until they are established for some weeks
+  allow_failures:
+
+  # Job: Clang tidy
+  - os: linux
+    env: V=1 PATH="/usr/bin:$PATH" NO_COMPILE=1 TEST=tidy
+    addons:
+      apt:
+        packages:
+        - elfutils
+        - libzstd1-dev
+        - libb2-dev
+    before_install:
+    - pip install --user compiledb
+
+  # Job: Check formatting
+  - os: linux
+    env: V=1 PATH="/usr/bin:$PATH" NO_COMPILE=1 TEST=check_format
+    addons:
+      apt:
+        packages:
+        - elfutils
+        - libzstd1-dev
+        - libb2-dev
```

Comment:
Since the jobs succeed as is, perhaps remove this part?

Reply:
I feel it's risky. Totally unrelated stuff will be unmergeable because of some small bug here. I would rather like to observe some pull requests after this one is merged and remove this in let's say two weeks.

Reply:
OK.